src/app/(layout)/posts/[slug]/_components/post-actions/ShareButton.tsx

```
@@ -1,28 +1,49 @@
+  };
   return (
     <button
-      onClick={handleShare}
+      onClick={handleClick}
       aria-label="공유하기"
       className="group flex flex-col items-center gap-1 cursor-pointer"
     >
       <span
         className={clsx(
-          "flex h-10 w-10 items-center justify-center rounded-full border transition-colors",
+          "flex h-9 w-9 items-center justify-center rounded-full border transition-colors",
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)